qemu/tests/transfer_file_over_ipv6.py

```diff
@@ -26,29 +29,12 @@ def run(test, params, env):
     port = params.get("file_transfer_port")
     password = params.get("password")
     username = params.get("username")
-    tmp_dir = params.get("tmp_dir", "/tmp/")
+    tmp_dir = params.get("tmp_dir")
```

Comment:
better to give a default value, or if you're sure that this parameter definitely have been set in cfg file, you can use params["tmp_dir"]

Reply:
This option has been definitely set in it's cfg file. Thanks for review.

Reply:
use params["tmp_dir"] instead of .get()